src/views/map/pokespawn.js

```diff
@@ -0,0 +1,61 @@
+@@ -0,0 +1,60 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import moment from 'moment'
+import pokedex from '../../models/pokedex.js'
+
+const PokemonSpawn = React.createClass({
+  getInitialState: function() {
+    return {
+    	expiration_time: this.props.spawn.expiration_time * 1000,
+    	endOfLife: {
```

Comment:
Is there a better way to get a reactive countdown? This works but probably isn't the best way to solve this issue.